src/main/java/com/dtforce/spring/kubernetes/SelectorEnabledDiscoveryClient.java

```diff
@@ -0,0 +1,17 @@
+package com.dtforce.spring.kubernetes;
+
+import org.springframework.cloud.client.ServiceInstance;
+import org.springframework.cloud.client.discovery.DiscoveryClient;
+
+import java.util.List;
+import java.util.Map;
+
+public interface SelectorEnabledDiscoveryClient extends DiscoveryClient
+{
+
+	// Equality-based matching
+	List<ServiceInstance> selectInstances(Map<String, String> match);
```

Comment:
As this is an interface, I would appreciate javadoc comments of both methods, including description of parameters.